### ISSUE TITLE
Evaluate the consequences of loosening fish distance critera

### ISSUE THREAD

**Issue by limnoliver** [USGS-R/delaware-model-prep#35]
While figuring out NGWOS data (#29), I loosened the fish distance criteria from 5k to 20k. We use the absolute value, such that we're not distinguishing between whether a site is coming from upstream or downstream of the end of the segment. Should we change our criteria based on upstream downstream? I don't _think_ so, but wanted to flag this as a potential issue. Here's an example of two reaches that have multiple sites that have big fish distance. 

![image](https://user-images.githubusercontent.com/15788176/81201889-6f5fd900-8f8b-11ea-9fe7-1f8d04dbbfd1.png)

**Comment by jzwart** [comment]
One potential issue with absolute values for fish distance would be if there is a confluence immediately downstream of the end of a segment - sites downstream of the confluence might not represent end of the segment very well.

**Comment by limnoliver** [comment]
Yes, good point Jake, and I would bet that many of the points of interest that were used to delineate reaches in the geospatial fabric were confluences.
